pegasus/rake/seed.rake

```diff
@@ -200,97 +200,15 @@ end
 $gdrive_ = nil
 
 namespace :seed do
-  def csv_smart_value(value)
-    return true if value == 'TRUE'
-    return false if value == 'FALSE'
-    return Date.strptime(value.to_s.strip, '%m/%d/%Y') if value.to_s.strip =~ /^\d{1,2}\/\d{1,2}\/\d{4}$/
-    return value
-  end
-
-  def import_csv_into_table(path, table)
```

Comment:
I'm a bit surprised this isn't used for i18n sheet that's left

Reply:
I believe the i18n sheet in the Data directory is converted into a YML file instead of being imported from CSV into the database:

https://github.com/code-dot-org/code-dot-org/blob/572fbb2a5dc84e130c0fb7ea1eaf46da2e44513e/pegasus/rake/i18n.rake#L55

Reply:
ah, you're right 👍